Overview.bs

```diff
@@ -285,38 +285,44 @@ in [[open-type/otff#data-types]]. As with the rest of OpenType, all fields use "
 URI Templates {#uri-templates}
 ------------------------------
 
-URI templates [[!rfc6570]] are used to convert numeric indices into URIs where patch files are located. Given a numeric index value,
-several variables are defined which are used to produce the expansion of the template:
+URI templates [[!rfc6570]] are used to convert numeric or string IDs into URIs where patch files are located.
+A string ID is a sequence of [[!unicode|Unicode]] code point values obtained from decoding a
+[[UTF-8]] string (see [[rfc6570#section-1.6]]). Several variables are defined which are used to produce the
+expansion of the template:
 
 <table>
   <tr><th>Variable</th><th>Value</th></tr>
   <tr>
     <td>id</td>
     <td>
-      The numeric index value converted to a string in hexadecimal representation (using the digits 0-9, A-F). No padding with 0's is
-      used.
+      If the input id is numeric then this is the  numeric value converted to a string in hexadecimal representation
+      (using the digits 0-9, A-F). No padding with 0's is used. Otherwise, this is the string id value.
     </td>
   </tr>
   <tr>
-    <td>d1</td><td>The first digit (least significant digit) of the numeric index in hexadecimal.</td>
+    <td>d1</td><td>The last character of the string in the id variable.</td>
   </tr>
   <tr>
-    <td>d2</td><td>The second digit of the numeric index in hexadecimal, or 0 if the index does not have a second digit</td>
+    <td>d2</td><td>The second last character of the string in the id variable.</td>
   </tr>
   <tr>
-    <td>d3</td><td>The third digit of the numeric index in hexadecimal, or 0 if the index does not have a third digit</td>
+    <td>d3</td><td>The third last character of the string in the id variable.</td>
   </tr>
   <tr>
-    <td>d4</td><td>The fourth digit of the numeric index in hexadecimal, or 0 if the index does not have a fourth digit</td>
+    <td>d4</td><td>The fourth last character of the string in the id variable.</td>
```

Comment:
Seems like we still need to say something about the behavior of d1-4 if the id is shorter than four characters. Either 0 or underscore seem fine to me. I guess the empty string is another option, but seems riskier. If you change from 0 you'll need to change the example below.

Reply:
Right, good catch. I'll make some updates.